jerry-core/api/jerry-snapshot.c

```diff
@@ -1198,157 +1198,6 @@ update_literal_offsets (uint8_t *buffer_p, /**< snapshot buffer start */
   while (buffer_p < buffer_end_p);
 } /* update_literal_offsets */
 
-#endif /* JERRY_ENABLE_SNAPSHOT_SAVE */
-
-/**
- * Merge multiple snapshots into a single buffer
- *
- * @return length of merged snapshot file
- *         0 on error
- */
-size_t
-jerry_merge_snapshots (const uint32_t **inp_buffers_p, /**< array of (pointers to start of) input buffers */
```

Comment:
Why this function is reordered?

Reply:
Because some static functions is used in jerry_merge_snapshots.

Reply:
I suspect the logic of merging and snapshot saving is combined in this patch. It should not.